configure-ironic.sh

```diff
@@ -46,6 +51,36 @@ connection = mysql+pymysql://ironic:${MARIADB_PASSWORD}@localhost/ironic?charset
 http_url = http://${IRONIC_URL_HOST}:${HTTP_PORT}
 fast_track = ${IRONIC_FAST_TRACK}
 
+EOF
+
+if [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
+    crudini --merge /etc/ironic/ironic.conf <<EOF
+[inspector]
+endpoint_override = https://${IRONIC_URL_HOST}:5050
+power_off = ${INSPECTOR_POWER_OFF}
+$([ ! -z "$CACERT_FILE" ] && echo "cafile = $CACERT_FILE")
```

Comment:
I think it's the right time to migrate to jinja for ironic.conf

Reply:
agreed, let me rework this PR